examples/supply_chain/rl/forecaster/oracle_forecaster.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import datetime
+import os
+
+import pandas as pd
+
+from .base_forecaster import BaseForecaster
+
+
+class OracleForecaster(BaseForecaster):
+    """
+    The oracle forecaster load data from form files. Each form needs to include the following columns.
+    FacilityName: facility name, should be defined in config.yml
+    SkuName: sku name, should be defined in config.yml
+    Date: date
+    Price: sku's mean price in this facility
```

Comment:
what's the meaning?
price to selling to downstream (downstream facilities or end-customers)?
or sth. else?